app/test/e2e/dataset-get-includes.spec.js

```diff
@@ -265,54 +266,51 @@ describe('Get datasets with includes tests', () => {
         const fakeDatasetTwo = await new Dataset(createDataset('cartodb')).save();
         const fakeDatasetThree = await new Dataset(createDataset('cartodb')).save();
 
-        nock(process.env.CT_URL)
-            .post(`/auth/user/find-by-ids`, {
-                ids: [fakeDatasetOne.userId, fakeDatasetTwo.userId, fakeDatasetThree.userId]
-            })
-            .reply(200, {
-                data: [{
-                    _id: fakeDatasetOne.userId,
-                    provider: 'local',
-                    name: 'test user',
-                    email: 'user-one@control-tower.org',
-                    role: 'USER',
-                    extraUserData: {
-                        apps: [
-                            'rw',
-                            'gfw',
-                            'gfw-climate',
-                            'prep',
-                            'aqueduct',
-                            'forest-atlas'
-                        ]
-                    }
-                }, {
-                    _id: fakeDatasetTwo.userId,
-                    role: 'MANAGER',
-                    provider: 'local',
-                    email: 'user-two@control-tower.org',
-                    extraUserData: {
-                        apps: [
-                            'rw',
-                            'gfw',
-                            'gfw-climate',
-                            'prep',
-                            'aqueduct',
-                            'forest-atlas'
-                        ]
-                    }
-                }, {
-                    _id: fakeDatasetThree.userId,
-                    role: 'MANAGER',
-                    provider: 'local',
-                    name: 'user three',
-                    extraUserData: {
-                        apps: [
-                            'rw'
-                        ]
-                    }
-                }]
-            });
+        createMockUser([{
+            id: fakeDatasetOne.userId,
```

Comment:
Relationship service relies on `_id` to map the users (here: https://github.com/resource-watch/dataset/blob/9d9927ebb2173b1a2dae20c6d73deeee13820b96/app/src/services/relationships.service.js#L97). Gonna check if I can rely just on that

Reply:
I can rely only on `_id` as long as the test users have the `_id` (which means I need to add `id` and `_id` to the users in the `test.contants.js` file). 

Not the best solution, but I think it's better to duplicate the ids there rather than on a per test basis. Let me know what you think.